src/data/alfajores/tokens-info.json

```diff
@@ -228,5 +228,12 @@
     "imageUrl": "https://raw.githubusercontent.com/valora-inc/address-metadata/main/assets/tokens/stCELO.png",
     "name": "Staked Celo",
     "symbol": "stCELO"
-  }
+  },
```

Comment:
I am also seeing the same address used for Alfajores and Mainnet. The Mainnet address looks correct; however, the Alfajores one should be changed or removed.